### DISCUSSION TITLE
When 12.5.0 will be avail?

### DISCUSSION THREAD

**Discussion by Mark-Muc** [benzino77/tasmocompiler#349] [Q&A]
https://github.com/arendst/Tasmota/blob/development/CHANGELOG.md#released
Thx Mark

**Comment by benzino77** [comment]
To answer the question "when" - I would say - when I find time ;)
I just push new version which supports Tasmota v12.5.0.

**Reply by Mark-Muc** [reply]
Many THX to u .. 
Mark
